packages/graphql-codegen-cli/src/load.ts

```diff
@@ -77,11 +77,29 @@ export async function loadDocuments(
   documentPointers: UnnormalizedTypeDefPointer | UnnormalizedTypeDefPointer[],
   config: Types.Config
 ): Promise<Types.DocumentFile[]> {
-  const loaders = [new CodeFileLoader(), new GitLoader(), new GithubLoader(), new GraphQLFileLoader()];
+  const loaders = [
+    new CodeFileLoader({
+      pluckConfig: {
+        skipIndent: true,
+      },
+    }),
+    new GitLoader(),
+    new GithubLoader(),
+    new GraphQLFileLoader(),
+  ];
+
+  const ignore: Array<string> = [];
+  for (const generatePath of Object.keys(config.generates)) {
+    if (extname(generatePath) === '') {
```

Comment:
What about directories? Shouldn't we add that as well?

Reply:
extname will always return an empty string if a path has no extension, theoretically a path with extension could also be a folder, in order to be 100% sure we would have to access the file-system here